dog-blog.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="zxx">
+
+<head>
+  <meta charset="utf-8">
+  <title>Adopt a Dog</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="styles/normalize.css">
+  <link rel="stylesheet" type="text/css" href="styles/styles.css">
+</head>
+
+<body>
+  <header>
+    <!--<div class="cart-checkout">
+      <div class="cart-box">
+        <img class="svg-cart" src="images/cart-icon.svg" alt="checkout cart icon">
+        <label class="price"> $1234.56</label>
+      </div>
+      <div>
+        <input class="cart-btn" type="submit" value="Checkout">
+      </div>-->
+    <div class="dog-img-header">
+      <img class="svg-dog-header" src="images/adoptadog-logo.svg" alt="Adopt-a-dog logo">
+      <p>Seattle's premiere dog adoption agency</p>
+    </div>
+  </header>
+  <nav class="nav-head">
+    <a class="nav-item-head" href="index.html">Home<span class="space">/</span></a>
+    <a class="nav-item-head" href="dog-listing.html">Listing<span class="space">/</span></a>
+    <a class="nav-item-head" href="dog-blog.html">Blog<span class="space">/</span></a>
+    <a class="nav-item-head" href="checkout.html">Checkout</a>
+  </nav>
+  <h1 class="blog-adoptadog-h1">Adoptadog Blog</h1>
+
+  <main>
+    <section class="container-blog">
+      <article class="blog-item">
```

Comment:
I would use flexbox, and specifically flex-wrap for your blog items. So on small screens, your text wraps below the image. Here's some material on flex-wrap: https://css-tricks.com/almanac/properties/f/flex-wrap/#:~:text=The%20flex%2Dwrap%20property%20is,new%20lines%20are%20stacked%20in.